openpype/hosts/maya/api/pipeline.py

```diff
@@ -681,6 +685,118 @@ def before_workfile_save(event):
         create_workspace_mel(workdir_path, project_name)
 
 
+def display_warning(message, show_cancel=False):
```

Comment:
Display warning doesn't really hint that it's to request an accept or cancel from the end user. The "Show feedback to user" also doesn't explain that in the docstring either. This is more of a prompt than a display.

Reply:
Changing name.